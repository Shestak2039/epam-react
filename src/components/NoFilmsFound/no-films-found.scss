@import '../../styles/colors';

.no-films-found-title {
  margin-top: 30px;
  
  text-align: center;
  
  color: $default-text-color;
}
