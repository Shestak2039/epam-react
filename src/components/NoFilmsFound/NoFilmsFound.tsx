import React from 'react';

import './no-films-found.scss';

const NoFilmsFound: React.FunctionComponent = () => (
    <h2 className="no-films-found-title">No film found</h2>
);

export default NoFilmsFound;
