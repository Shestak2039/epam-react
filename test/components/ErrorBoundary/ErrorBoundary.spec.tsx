import React from 'react';
import { render } from '@testing-library/react';
import ErrorBoundary from '../../../src/components/common/ErrorBoundary/ErrorBoundary';
import NoFilmsFound from '../../../src/components/NoFilmsFound/NoFilmsFound';


describe('ErrorBoundary', () => {
    test('renders ErrorBoundary snapshot', () => {
        const { asFragment } = render(<ErrorBoundary><NoFilmsFound /></ErrorBoundary>);

        expect(asFragment()).toMatchSnapshot();
    });
});
