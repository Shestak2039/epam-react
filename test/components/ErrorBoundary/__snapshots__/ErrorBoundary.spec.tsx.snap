// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary renders ErrorBoundary snapshot 1`] = `
<DocumentFragment>
  <h2
    class="no-films-found-title"
  >
    No film found
  </h2>
</DocumentFragment>
`;
